src/apis/PostsEndpoint.js

```diff
@@ -52,7 +52,7 @@ const formatPostData = (data)=> {
     sendreplies: data.sendreplies,
     sr: data.sr,
     iden: data.iden,
-    captcha: data.captcha,
+    'g-recaptcha-response': data.gRecaptchaResponse,
```

Comment:
Yep, it was used to identify whether the recaptcha need show or not. Will send another PR of reddit-mobile soon.